kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/core/utils.kt

```diff
@@ -0,0 +1,117 @@
+package space.kscience.kmath.tensors.core
```

Comment:
probably better to create `internal` pacage for internals. It is a common practice.